ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-ref.yaml

```diff
@@ -36,6 +36,12 @@ ref:
   - name: OPERATOR_UPDATE_CHANNEL
     default: "stable"
     documentation: "The operator subscription update channel"
+  - name: PODVM_IMAGE_URL
+    default: ""
+    documentation: |
+      By the default the OSC operator will build a podvm image at the time kataconfig is
+      deployed, however, you can override it by passing the URL from where it can
+      download an already built image.
```

Comment:
Is there a public gallery somewhere or do you have to build it and copy to the current run's account somehow?

Reply:
Hi @ldoktor!

You have to build the disk image, put in container image and push to registry that accessible from your cluster installation. The image builder will then pick up that image and publish in a gallery.

There is a documentation here but it's a bit outdated: https://github.com/openshift/sandboxed-containers-operator/tree/devel/config/peerpods/podvm#podvm-image-upload-configuration

Reply:
Hi @tbuskey , afaik  we don't have a complete and up-to-date documentation on that. I suggest to keep this parameter's doc as is, raise a JIRA and hopefully someone on dev will pick it to work on?